### ISSUE TITLE
Alpha complete, initial test successful!

### ISSUE THREAD

**Issue by VTransRickScott** [VTrans-Rail/Project-Web-GIS#7]
Hi Stephen,

I've complete all the steps for the post-processing, and have successfully tested initial build followed by post process.  Next step is to talk about next steps.  :-)

Cheers!
Rick

**Comment by srsjojo** [comment]
Dude! You rock! :100: :100: :100:

We should get together mid to late next week to go over where we're at, but here are some thoughts for now:

- Is the system fully setup so that Josh can add a new project and a project manager can input a status on it? What does that look like? (nightly, hourly, on-demand, or self-service?)
- When a project gets added, does the new project name propagate into the domain?
- How do the status updates make it back into the database? (nightly, hourly, on-demand, or self-service?)
- How can the PMs see their old updates? (this is fairly trivial, but needs to be done)
- If they want to see their past updates before they're committed what then?
- What about when a project is done? How does Josh remove it from the system? Is it removed from the domain? 

Basically (pronoun) will come up with what we expect this process to look like and we'll check our progress against that model and figure out what else needs done.

Thanks again! :+1:

**Comment by VTransRickScott** [comment]
My items for meeting agenda:
Walkthrough and demo process for Stephen

Discuss rollout to production
	What goes/points to where?
	What will initial testing look like?
		Who is test what?
		What will success look like?
		What will failure look like?
		How long do we want to run the initial testing phase?
	How often do you want to run post-process?
	Beta testing
		Who is test what?
		What will success look like?
		What will failure look like?
		How long do we want to run the beta testing phase?
	Phase 2 scope (Reporting, etc.[See "Overall Features Needed" in scripts-doc.md])

Do you have other items?  Which of us shall schedule the meeting (ideally a room with a computer)?

**Comment by srsjojo** [comment]
My number 1 priority this week is getting materials ready for the legislature. Once that's taken care of I'll get something scheduled with you.

**Comment by VTransRickScott** [comment]
No sweat, just didn't want to lose track of my thoughts  :-)

**Comment by srsjojo** [comment]
Per discussion, next steps:

[ ] Rick points scripts to production
[ ] Stephen brings backup load table to current
[ ] Rick update SDE connections file
[ ] Rick send Stephen connection text file
[ ] Stephen runs alpha test with Consultants + PM's

**Comment by srsjojo** [comment]
I'm looking into the system now and I'm not seeing the changes. What should I be looking for? Will all the changes take place when it "runs" tonight?

**Comment by VTransRickScott** [comment]
Correct, changes should take place tonight, after I run the script.  You should see them tomorrow.  I will do a rough comparison before/after, so I can point you in the right direction for QA/QC.
